src/nanoemoji/extract_svgs_from_otsvg.py

```diff
@@ -77,14 +77,13 @@ def main(argv):
         svg.svg_root.append(svg_defs)
         svg.svg_root.append(svg_g)
 
-        for gid in range(min_gid, max_gid + 1):
-            glyph_name = font.getGlyphName(gid)
-            width, _ = metrics[glyph_name]
-            svg.svg_root.attrib["viewBox"] = f"0 0 {width} {height}"
-            dest_file = out_dir / f"{gid}.svg"
-            with open(dest_file, "w") as f:
-                f.write(svg.tostring(pretty_print=True))
-            logging.debug("Wrote %s", dest_file)
+        glyph_name = font.getGlyphName(gid)
+        width, _ = metrics[glyph_name]
+        svg.svg_root.attrib["viewBox"] = f"0 0 {width} {height}"
+        dest_file = out_dir / f"{gid}.svg"
```

Comment:
maybe pad to 5 leading zeros (to fit max 0xFFFF maxp.numGlyphs) so that it's easier to browse in a directory when sorted alphanumerically?

```diff
- dest_file = out_dir / f"{gid}.svg"
+ dest_file = out_dir / f"{gid:05d}.svg"
```

Reply:
or pad with the least number of zeros required to fit the maximum gid, e.g.

```python
ndigits = int(math.log10(max(gids))) + 1
dest_file = out_dir / f"{str(gid).zfill(ndigits)}.svg"
```

Reply:
Will change when I escape meeting purgatory